lona/static_file_loader.py

```diff
@@ -228,3 +214,15 @@ def resolve_path(self, path: str) -> Optional[str]:
         logger.debug("'%s' was not found", path)
 
         return None
+
```

Comment:
Why are there two functions with new naming outside of the class now? Also why is the first function named class_tree but returns a list of types?

Reply:
Fixed by removing `class_tree` function

Reply:
The naming is still odd. And why has it to be outside the class now?

Reply:
> The naming is still odd.

Are you talking about `def all_subclasses`? What's wrong with it?

> And why has it to be outside the class now?

I'd ask why it should be inside class if it isn't related to class at all? It's a separate (and pure) function which can be used in any class/function.

Reply:
All other functions like that in the project start with ``get_``.

Yes its not inherently related to the class, but if we decide to make it an standalone helper it shouldn't be stored in this module. Maybe ``lona.imports``?

Reply:
Good point. Fixed.